customScripts/expansiontravel.gd

```diff
@@ -48,7 +52,11 @@ func getzonetraveltext(zone,progress):
 		#if !globals.state.backpack.stackables.has('bandage') && i.health < i.max_health/2:
 		#	text =
 		#	break
-	if rand_range(0,1) > 0.5:
+	if rand_range(0,1) > 0.9:
+		array.append_array(["A flock of crows pick at a well scavenged corpse nearby.  Upon examination of the nearly stripped bones and tattered remains of clothing, you figure this poor traveler was likely a poor sword for hire or vagabond.  It's unclear what killed him.  Whatever possessions he might have once had have long been looted or dragged away by vermin.  Unable to determine even their place of origin or likely destination, you decide to move on."])
```

Comment:
Both of these entries do not need append_array() as they contain only a single string

Reply:
comment incorporated on new commit